File: programacao/c/respostas/operacoes/25.c
#include <stdio.h>

int main() {
    int total_eleitores, votos_brancos, votos_nulos;
    double percentual_brancos, percentual_nulos;

    printf("Digite o total de eleitores: ");
    scanf("%d", &total_eleitores);

    printf("Digite o total de votos brancos: ");
    scanf("%d", &votos_brancos);

    printf("Digite o total de votos nulos: ");
    scanf("%d", &votos_nulos);

    percentual_brancos = ((double) votos_brancos / total_eleitores) * 100;
    percentual_nulos = ((1.0 * votos_nulos) / total_eleitores) * 100;

    printf("Percentual de votos brancos: %.2f%%\n", percentual_brancos);
    printf("Percentual de votos nulos: %.2f%%\n", percentual_nulos);

    return 0;
}

File: programacao/python/respostas/operacoes/25.py
total_eleitores = int(input("Total de eleitores: "))
votos_brancos = int(input("Total de votos brancos: "))
votos_nulos = int(input("Total de votos nulos: "))

percentual_brancos = votos_brancos / total_eleitores * 100
percentual_nulos = votos_nulos / total_eleitores * 100

print(f"Percentual de votos brancos: {percentual_brancos:.2f}%")
print(f"Percentual de votos nulos: {percentual_nulos:.2f}%")
